scripts/pantheon/deploy

```diff
@@ -41,11 +41,11 @@ then
   if [[ "$CANONICAL_ENV" != "dev" ]]
   then
     terminus -n env:clone-content "$TERMINUS_SITE.$CANONICAL_ENV" dev
-    if [[ ! -z "$SANATIZE_SCRIPT" ]]
+    if [[ ! -z "$SANITIZE_SCRIPT" ]]
     then
-      echo "Sanatizing Database."
-      # If we've set a sanatization script for the database, run it.
-      $( $SANATIZE_SCRIPT )
+      echo "Sanitizing Database."
+      # If we've set a sanitization script for the database, run it.
+      $( $SANITIZE_SCRIPT )
```

Comment:
This COULD be a breaking change, we may have to create a compatibility layer in set-environment.

Reply:
We can def. work together on this. I honestly would not know where to begin.

Reply:
Moved this to it's own issue/PR.